tests/integration/support/utils.go

```diff
@@ -77,3 +77,29 @@ func parseTemplate(s string, config pkg.ProjectConfig) string {
 	}
 	return parsedString
 }
+
+func compareStringSlice(x, y []string) bool {
```

Comment:
Did we copy this from somewhere? If so we need proper attribution for this code

Reply:
This was copied from stackoverflow.

Reply:
Ok I'm that case just include the link to the stack overflow post right above this so we know where it came from

Reply:
done.

Reply:
@yasirfolio3 I don't see the link, did you forget to commit 😄

Reply:
done, sorry about the confusion, i actually thought you asked me to comment the link here, apologies 👍